airflow/contrib/hooks/ssh_hook.py

```diff
@@ -171,6 +171,13 @@ def get_conn(self) -> paramiko.SSHClient:
                              'against Man-In-The-Middle attacks')
             # Default is RejectPolicy
             client.set_missing_host_key_policy(paramiko.AutoAddPolicy())
+
+        # restart host_proxy to avoid "Broken Pipe" error
+        if getattr(self, 'host_proxy') is not None:
+            if not self.host_proxy.closed:
+                self.host_proxy.close()
```

Comment:
HostProxy (Paramiko's ProxyCommand) is basically a subprocess. Closing of proxy socket does not mean subprocess is closed. This is for making sure subprocess is also closed.